src/Tizen.NUI/src/public/BaseComponents/ImageView.cs

```diff
@@ -491,6 +490,7 @@ public string ResourceUrl
             }
             set
             {
+                _url = (value == null? "" : value);
```

Comment:
Nothing happen when assign a null value. is it intended?
let's see 494 line, it will called with value(null)
``` c#
SetValue(ResourceUrlProperty, value);
```
and 
`propertyChanged` callback was called with newValue(null)

and it ignored (see 37 line)
`if (newValue != null)`

Reply:
And this behavior is not matched with summary `When it is set as null, it gives empty string ("") to be read.`
when a null value was assigned, will get null

Reply:
How about change defaultValueCreator callback of ResourceUrl property?
It is wrong in #538 PR. 
It returing null with default value. 
It will be issue